migration/migrator/migrations/course/20230921190909_add_previous_rotating_section_trigger.py

```diff
@@ -0,0 +1,58 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+        ALTER TABLE users -- Create column
+        ADD COLUMN previous_rotating_section INTEGER;
+
+        -- Create empty trigger function that is replaced by new trigger function file
+        CREATE OR REPLACE FUNCTION public.update_previous_rotating_section()
+            RETURNS trigger
+            LANGUAGE plpgsql
+            AS $$
+        BEGIN
+            RETURN NEW;
+        END;
+        $$;
+
+        -- Attatch trigger function
+        CREATE TRIGGER before_update_users_update_previous_rotating_section
+        BEFORE UPDATE ON public.users
```

Comment:
This will only need to get run if the rotating section changes, right? If so, can we change it to `BEFORE UPDATE OF rotating_section ON public.users`?